dist/profile/manifests/confluence.pp

```diff
@@ -94,31 +94,11 @@
     restart_service => true,
   }
 
-  ### to put maintenance screen up, comment out the following and comment in the apache::vhost for https://jenkins-ci.org
-  ### #if
-  #file { '/etc/apache2/sites-enabled/25-wiki.jenkins-ci.org.conf':
-  #  ensure => 'link',
-  #  target => '/etc/apache2/sites-available/wiki.jenkins-ci.org.maintenance.conf',
-  #}
-  ### #else
-  apache::vhost { 'wiki.jenkins-ci.org':
-    port            => '443',
-    docroot         => '/srv/wiki/docroot',
-    access_log      => false,
-    error_log_file  => 'wiki.jenkins-ci.org/error.log',
-    log_level       => 'warn',
-    custom_fragment => template("${module_name}/confluence/vhost.conf"),
-
-    notify          => Service['apache2'],
-    require         => File['/var/log/apache2/wiki.jenkins-ci.org'],
-  }
-  ### #endif
-
   # We can only acquire certs in production due to the way the letsencrypt
   # challenge process works
   if (($::environment == 'production') and ($::vagrant != '1')) {
     letsencrypt::certonly { 'wiki.jenkins.io':
-        domains     => ['wiki.jenkins.io'],
+        domains     => ['wiki.jenkins.io','wiki.jenkins-ci.org'],
```

Comment:
My only concern is that this won't trigger a certificate renewal automatically. I'm making a mental note to check that when we push this to production